grudge/models/wave.py

```diff
@@ -198,27 +198,28 @@ class VariableCoefficientWeakWaveOperator(HyperbolicOperator):
 
     The sign of :math:`v` determines whether we discretize the forward or the
     backward wave equation.
-
-    :math:`c` is assumed to be constant across all space.
     """
 
-    def __init__(self, dcoll, c, source_f=None,
+    def __init__(self, actx, dcoll, c, source_f=None,
             flux_type="upwind",
             dirichlet_tag=BTAG_ALL,
             dirichlet_bc_f=0,
             neumann_tag=BTAG_NONE,
             radiation_tag=BTAG_NONE):
+        """
+        :arg c: a thawed (with *actx*) :class:`~meshmode.dof_array.DOFArray`
+            representing the propogation speed of the wave.
```

Comment:
Should follow the advice from https://github.com/inducer/arraycontext/pull/22/: Just use the array context from `c`.